packages/create-plugin/README.md

```diff
@@ -99,6 +99,18 @@ For more information see [here](https://grafana.com/developers/plugin-tools/migr
 
 You can read more about customizing or extending the basic configuration [here](https://grafana.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/extend-configurations)
 
+
+```bash
+# Run this command from the root of your plugin
+cd ./my-plugin
+
+npx @grafana/create-plugin@latest provisioning
+```
+
+For more information see [here](https://grafana.com/developers/plugin-tools/create-a-plugin/)
```

Comment:
Is this the correct link? it points to the landing page of create-plugin and there's no information there about provisioning.